src/test/ui/numbers-arithmetic/promoted_overflow.rs

```diff
@@ -3,6 +3,8 @@
 // run-fail
 // error-pattern: overflow
 // compile-flags: -C overflow-checks=yes
+// for some reason, fails to match error string on wasm32-unknown-unknown with stripped debuginfo and symbols
+// compile-flags:-Cstrip=none
```

Comment:
Fails to find error string on `wasm32-unknown-unknown` when stripped, even if error string didn't contain backtrace, idk why.